API/NoDocs/Aspire.Azure.Search.Documents/AzureSearchSettings.cs

```diff
@@ -0,0 +1,18 @@
+// Assembly 'Aspire.Azure.Search.Documents'
+
+using System;
+using System.Runtime.CompilerServices;
+using Aspire.Azure.Common;
+using Azure.Core;
+
+namespace Aspire.Azure.Search.Documents;
+
+public sealed class AzureSearchSettings : IConnectionStringSettings
```

Comment:
Nit - Worth calling out that this pattern varies from the default of `<Namespace>Settings`.  I think this reads better than `AzureSearchDocumentsSettings` though and we should keep it.